### PR TITLE
feat(web): custom stylesheets

### PR DESCRIPTION
**Pull request by Wingysam**
This draft pull request is a work-in-progress at implementing a Custom CSS setting to the system settings page.
image

### CODE REVIEW

File: web/src/routes/admin/system-settings/+page.svelte [hunk 1]
```diff
@@ -110,6 +111,13 @@
       >
         <FFmpegSettings disabled={$featureFlags.configFile} ffmpegConfig={configs.ffmpeg} />
       </SettingAccordion>
+
+      <SettingAccordion
```
Comment (martabal) [inline web/src/routes/admin/system-settings/+page.svelte:115]: Can you place it in the first position (settings are sorted in alphabetical order)?

**Comment by martabal** [comment]
Can't wait to see custom themes :sunglasses:

File: web/src/routes/admin/system-settings/+page.svelte (hunk 1 above)
Comment (Wingysam) [reply web/src/routes/admin/system-settings/+page.svelte:115]: Yes I can!

**Comment by Wingysam** [comment]
All functionality is working as intended and all web/server/e2e tests are passing locally. Ready for review!

**Review by jrasm91 — APPROVED** [review]
Lgtm

**Comment by etnoy** [comment]
Haven't checked the code itself, but love to see the theming support!

**Review by jrasm91 — APPROVED** [review]
This looks good, thanks for working on it. 

Everything is great, except I would have preferred different names for stuff, like:
- Admin Settings > Theme (or Theme Customizations or Custom Theme) 
- `theme.customCss`
- `THEME_CUSTOM_CSS`
- `theme-settings.svelte`
- etc.

Stylesheet is very specific and exactly what you implemented, but I think a more general "theme" naming would have been more appropriate.

I am OK merging it as is, if you do not want to change it. If more related changes are suggested in the future, I will require it at that time. Let me know if this is something you would like to address now, if not I will merge it as is.

**Comment by Wingysam** [comment]
@jrasm91 I've made the requested changes.

**Comment by Wingysam** [comment]
Oops, forgot to update the server test and run the tests for the server. Fixed that and tests pass now.

**Comment by jrasm91** [comment]
> @jrasm91 I've made the requested changes.

Thank you so much :pray:

**Pull request merged**